org.serioussam.SeriousSamClassic-VK.yaml

```diff
@@ -0,0 +1,118 @@
+app-id: org.serioussam.SeriousSamClassic-VK
```

Comment:
is `serioussam.org` really the website for this game?

Reply:
No. No. I just took other projects as examples.
There is only source code on GitHub

Reply:
So when you checked "yes" here you lied?

![image](https://github.com/flathub/flathub/assets/114441/2cdd2379-7610-4467-a35c-c598c37b03f4)

This app id does not meet the requirement. you MUST change it.

Reply:
Be careful what you say about lying. This is inattention on my part. English is not my native language. Everyone makes mistakes sometimes.If I hadn't been asked, I wouldn't have made a flatpack at all.